base/src/main/java/com/mindera/skeletoid/analytics/IAnalyticsManager.kt

```diff
@@ -66,7 +66,7 @@ interface IAnalyticsManager {
     fun trackPageHit(
         activity: Activity,
         screenName: String,
-        screenClassOverride: String
+        screenClassOverride: String? = null
```

Comment:
Cant we make this non null ?

Reply:
We were using null in several places. And I'm still not sure if non null is the behaviour we actually want. This is supposed to be used to override, what if we don't want to override?

Reply:
I guess the idea here is for the override to be optional. For that you need a default value that you can check and ignore. Since theoretically "empty string" is a valid value, while null is not, I think it makes sense to use a nullable string. That said, I don't see a use case where you would want to override the screenClass with an empty string, so keeping it non-nullable and checking the string for empty probably won't break the world

Reply:
@pedro-belchior Do you agree with this behaviour ☝️ 
The default would change to empty string that would function as null now does.